front/app/components/admin/ActionForm/FlowVisualization/Blocks/SSOConfigModal.tsx

```diff
@@ -25,19 +25,25 @@ import { getVerifiedDataList } from './utils';
 interface Props {
   opened: boolean;
   verificationExpiry: number | null;
-  verificationMethodMetadata: ActionMetadata;
   onClose: () => void;
   onChangeVerificationExpiry: (value: number | null) => void;
 }
 
 const SSOConfigModal = ({
   opened,
   verificationExpiry,
-  verificationMethodMetadata,
   onClose,
   onChangeVerificationExpiry,
 }: Props) => {
   const localize = useLocalize();
+
+  const { data: verificationMethod } = useVerificationMethodVerifiedActions();
```

Comment:
Fetching this data in the component instead of having it passes as a prop, nothing exciting